src/common/utils/compare.ts

```diff
@@ -258,3 +273,78 @@ export function isCollegeCounty(location: Location) {
   const countyPopulation = location.population;
   return ftEnrollment && ftEnrollment / countyPopulation > threshold;
 }
+
+// For sharing:
+
+export function getShareQuote(
+  sorter: Metric,
+  countyTypeToView: MetroFilter,
+  sliderValue: GeoScopeFilter,
```

Comment:
I think this type is wrong.  It's not a `GeoScopeFilter`, it's just a `number` (0, 50, or 99).

But it might be better if it _was_ the `GeoScopeFilter` because that's a much clearer type (doesn't involve magic numbers) and then you wouldn't need to have the exported `sliderValueMap` thing.

Reply:
The name `sliderValue` is probably confusing things here-- the type is correct. `sliderValue` is the GeoScopeFilter that the numerical slider value (0, 50, 99) corresponds to.

As for the magic numbers themselves, the [material UI slider component](https://material-ui.com/api/slider/) calls for them unfortunately

Reply:
Oh just kidding actually. sliderValue is the magic #. hmmmm